src/clientdlg.cpp

```diff
@@ -1093,6 +1106,12 @@ void CClientDlg::OnTimerCheckAudioDeviceOk()
                                tr ( "Your sound card is not working correctly. "
                                     "Please open the settings dialog and check the device selection and the driver settings." ) );
     }
+    /*else
+    {
+      #if defined ( ANDROID ) || defined ( Q_OS_ANDROID )
+        QMessageBox::warning ( this, APP_NAME, pClient->Sound.getAvailableDevices() ); //DEBUG NGOCDH
+      #endif
+    }*/
```

Comment:
Still needed?

Reply:
Seems like a test at an earlier stage of the dev that I don't even remember lol. Will remove it.